manifests/w/WatchGuard/MobileVPNwithSSLclient/12.7.2/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient
+PackageVersion: 12.7.2
+PackageLocale: en-US
+Publisher: WatchGuard
+PackageName: WatchGuard Mobile VPN with SSL Client

```

```suggestion
PackageName: WatchGuard Mobile VPN with SSL client
```
This must match Control Panel.